apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx

```diff
@@ -7,11 +7,28 @@ import {
     processingStatusOptions
 } from './InvestigationFormTypes';
 import { SingleSelect, MultiSelect } from 'design-system/select';
-import { ConceptAutocomplete } from 'options/autocompete/ConceptAutocomplete';
+import { ConceptOptionsService } from 'generated';
+import { useEffect, useState } from 'react';
+import { Selectable } from 'options';
 
-const CriteriaSearchFields = () => {
+type Props = {
+    outbreakCodeSetNm?: string;
+};
+
+const CriteriaSearchFields = ({ outbreakCodeSetNm = 'OUTBREAK_NM' }: Props) => {
     const form = useFormContext<InvestigationFilterEntry, Partial<InvestigationFilterEntry>>();
 
+    const [outbreakNames, setOutbreakNames] = useState<Selectable[]>();
+
+    useEffect(() => {
+        ConceptOptionsService.conceptSearch({
+            name: outbreakCodeSetNm,
+            criteria: ''
+        }).then((response) => {
+            setOutbreakNames(response.options);
+        });
+    }, [outbreakCodeSetNm]);
```

Comment:
Question: `useOptions`?

Reply:
This use case is prevalent enough for a `ConceptMultiSelect` component to be created that takes the code set name as a parameter.